### PR TITLE
Assignment for Lab01

### PR DESCRIPTION
**Pull request by geoffreycc**

### CODE REVIEW

**Comment by meganwalter** [comment]
Hey - You want to make sure you're submitting a PR to the codefellows repo master - come see me after class and I'll show you.

File: JS/app.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+var articleContent = [];
+
+function ArticleObj(art) {
+  this.title = art.title;
+  this.imgName = art.imgName;
+  this.imgPath = 'Images/' + art.imgName + '.jpg';
```
Comment (sjschmidt44) [inline JS/app.js:7]: Lowercase files and folders are best practice!

File: JS/app.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  this.about = art.about;
+  this.link = art.link;
+  articleContent.push(this);
```
Comment (sjschmidt44) [inline JS/app.js:10]: Nice use of `articleContent.pus()` here in the constructor.

File: JS/app.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  $section.find('a').attr('href', this.link).text(this.title + ' Repository');
+
+  $section.removeClass('newArticle');
+  $section.addClass('articleContent');
+
+  return $section;
+};
+
+articles.forEach(function(ele){
+  var art = new ArticleObj(ele);
```
Comment (sjschmidt44) [inline JS/app.js:28]: You don't actually need to assign this to a variable here. You can just instantiate the new ArticleObj and let it do it's thing.

**Pull request merged**